ROMFS/px4fmu_common/init.d/rc.gnd_defaults

```diff
@@ -0,0 +1,36 @@
+#!nsh
+
+set VEHICLE_TYPE ugv
+
+if [ $AUTOCNF == yes ]
+then
+#
+# Default parameters for FW
+#
+	param set RTL_RETURN_ALT 100
+	param set RTL_DESCEND_ALT 100
+	param set RTL_LAND_DELAY -1
```

Comment:
In this case, and if you want to keep this, you need to refactor the Return-to-launch mode for ground vehicles, as it will be only done on XY plane, without takeoff and land of course. Have you tough of this?